test/lib_ic/test_ic_spec/test_ic_spec.py

```diff
@@ -113,8 +107,8 @@ class ICSpec(object):
 #             invert_check=['convergence']),
     TestCase('Short Echo', filters.Identity(), filters.ShortEcho()),
 #    TestCase('Zero at 5', filters.Identity(), filters.ZeroAt(5)),
-    TestCase('Moving source', filters.Identity(), filters.MovingSource(move_frequency=1.5),
-             dont_check=['stability']),
+#    TestCase('Moving source', filters.Identity(), filters.MovingSource(move_frequency=1.5),
```

Comment:
Can you add a comment referring to the issue where you've discussed commenting this?